examples/beam/without_job_server/beam_wordcount_java.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
         - name: beam-wordcount-java
-          image: apache/beam_java_sdk
+          image: apache/beam_java_sdk:2.23.0
           command: ["/bin/bash"]
           args:
           - apt-get update && apt-get install -y maven
```

Comment:
I tested with this example, but the job failed with `/bin/bash: apt-get update && apt-get install -y maven: No such file or directory`.